Frontend/Views/Transfers/IncomingTrust.cshtml

```diff
@@ -42,10 +42,10 @@
         <form method="get" asp-action="SearchIncomingTrust">
             <div class="govuk-form-group @formClasses">
                 <fieldset class="govuk-fieldset" aria-required="true">
-                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                        <h1 class="govuk-fieldset__heading">Add the incoming trust name</h1>
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                        <h1 class="govuk-fieldset__heading">Add the incoming trust name for @ViewData["OutgoingTrustName"]</h1>
```

Comment:
👀 This is the outgoing academy name, not the trust name

In the future we'll have multiple transferring academies, so having this in the title allows us to differentiate between which one we're adding the incoming trust for